shuttle/templates/loaded_views/add_user.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'shuttle_base.html' %}
+    <title>Shuttle | {% block page_title %}Add a New User{% endblock %}</title>
+
+{% block body_content %}
+    <div class="add-user-search">
+        <h4>Add a New User</h4>
+        <div class="new-user-text">
+            <input type="text" id="f-name" placeholder="First Name">
+            <input type="text" id="l-name" placeholder="Last Name">
+        </div>
+        <br>
+        <button type="button" class="btn btn-primary" data-toggle="modal" onclick="searchButton()">Search</button>
```

Comment:
Seems like the css isn't working quite right here

![Screen Shot 2020-07-15 at 1 06 19 PM](https://user-images.githubusercontent.com/26125143/87579793-1d7ab400-c69c-11ea-9f2b-6fe8aca9e1e6.png)

Reply:
What circumstances did this happen in?

Reply:
I mean I opened that page and that is how it looks. On a 2560 x 1440 screen